src/text/isMatch.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Test a string against regexp
+ * @param {string} target
+ * @param {string} regExp
```

Comment:
The parameter's name is `rex` in the code.

Reply:
fixed, now the parameter type is better handled, wil construct a RegExp from string or use the provided one